src/main/java/com/fasterxml/jackson/dataformat/xml/util/DefaultXmlPrettyPrinter.java

```diff
@@ -79,10 +79,24 @@
 
     /**
      * By default, will try to set as System.getProperty("line.separator")
```

Comment:
Can you add a '.' at the end of each sentence here?

Reply:
@pjfanning Got it